.github/workflows/code-pushup-fork.yml

```diff
@@ -34,5 +34,3 @@ jobs:
         run: npm ci
       - name: Run Code PushUp action
         uses: code-pushup/github-action@v0
```

Comment:
The current [_Code PushUp (fork)_ error](https://github.com/push-based/nx-verdaccio/actions/runs/15922559146/job/44912469836?pr=79) is due to old package version. I've tried to fix by updating dependencies, but it seems like the old version is still being run - notice [reported action inputs](https://github.com/push-based/nx-verdaccio/actions/runs/15922559146/job/44912469836?pr=79#step:5:19) don't match updated [workflow file](https://github.com/push-based/nx-verdaccio/actions/runs/15922559146/workflow?pr=79#L36) (old `bin` option).

I suspect this might be because of some GitHub security feature with regards to forks changing workflows :thinking: